docs/architecture/adr-007-universal-share-prefix.md

```diff
@@ -1,5 +1,9 @@
 # ADR 007: Universal Share Prefix
 
+## Status
+
+Implemented
```

Comment:
Implemented is not a real status according to ADR template. Do we want to add implemented as a status type, and if so, should we update all the implemented Accepted statuses to Implemented?

Reply:
> Do we want to add implemented as a status type

Sure.

> should we update all the implemented Accepted statuses to Implemented?

Will do in this PR.

Reply:
Resolved by https://github.com/celestiaorg/celestia-app/pull/1407/commits/ce7c3bebcd9fd762e9d635d437a15e07f6ed8710